concat-utils.php

```diff
@@ -1,6 +1,16 @@
 <?php
 
+if ( ! defined( 'VIP_CONCAT_MAX' ) ) {
+	define( 'VIP_CONCAT_MAX', 150 );
+}
+
```

Comment:
I think rather than introducing a new constant, providing a basic API would be better long-term. I.e. `set_group_size` and `get_group_size` or something similar.

Reply:
I'm not entirely sure what you mean here. (filters? helper functions?)

I added the constant initially because it seems like the nice thing to do, but maybe it's not needed at all. Currently there's no customization _and_ poor handling of large groups. This PR doesn't _have_ to offer both solutions.

Reply:
Pretty much helper functions with the same default value, rough pseudo:

```php
class WPCOM_Concat_Utils {
  $max = 150;
  function set_max_bucket_size()
  function get_max_bucket_size()
}
```

Reply:
Just noting to close the loop #77 also uses a hardcoded value.